erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -68,6 +68,13 @@ WebRtcConnection::~WebRtcConnection() {
   ELOG_DEBUG("%s message: Destructor ended", toLog());
 }
 
+void WebRtcConnection::close() {
```

Comment:
why do you need to change the location? it might conflict with an ongoing PR ;)

Reply:
No particular reason, it liked it better that way. I switched it back.